Client.xcodeproj/project.pbxproj

```diff
@@ -4145,8 +4150,6 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				APPLICATION_EXTENSION_API_ONLY = YES;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
 				CURRENT_PROJECT_VERSION = 1;
```

Comment:
Don't delete this (the signing parts of the project should remain unchanged). Same with remaining lines in this file.